translations/en_GB/atrium-translations-en_GB-common/src/main/kotlin/ch/tutteli/atrium/translations/DescriptionResultAssertion.kt

```diff
@@ -0,0 +1,11 @@
+package ch.tutteli.atrium.translations
+
+import ch.tutteli.atrium.assertions.DescriptiveAssertion
+import ch.tutteli.atrium.reporting.translating.StringBasedTranslatable
+
+/**
+ * Contains the [DescriptiveAssertion.description]s of the assertion functions which are applicable to [Map].
+ */
+enum class DescriptionResultAssertion(override val value: String) : StringBasedTranslatable {
+    COULD_NOT_INTERPRET_INPUT("$COULD_NOT_EVALUATE_DEFINED_ASSERTIONS unable to interpret/understand input \n$VISIT_COULD_NOT_EVALUATE_ASSERTIONS")
```

Comment:
good analysed, I guess you saw `CANNOT_EVALUATE_SUBJECT_EMPTY_ITERABLE` in DescriptionIterableAssertion. Should actually not be used any more. The hint was for `Assert` and is no longer true for `Expect`. It suffices if you have something like `IS_NOT_SUCCESS("!! is not a Success")` for en_GB and `IS_NOT_SUCCESS("!! ist kein Success")` for de_CH